assets/html/scss/documenter/components/_admonition.scss

```diff
@@ -10,7 +10,7 @@ $admonition-body-color: () !default;
 $admonition-header-background: () !default;
 $admonition-header-color: () !default;
 @each $name, $color in $admonition-colors {
-  $bg: lighten-color($color, 4);
+  $bg: lighten-color($color, 5);
```

Comment:
This change makes sure the default theme's admonitions all render with a dark background color, instead of the current behaviour:
![image](https://user-images.githubusercontent.com/6735977/118800615-65f89480-b8a0-11eb-94e5-745c50a905e5.png)
New:
![image](https://user-images.githubusercontent.com/6735977/118800641-701a9300-b8a0-11eb-8d8d-b36a1d1ffc5c.png)

Imho this improves legibility by a lot.